tests/utility.py

```diff
@@ -38,8 +42,59 @@ def get_docker_os_type():
 
 
 def get_platform_type():
-    if get_docker_os_type() == 'windows':
-        platform_type = 'windows-amd64'
+    if get_docker_os_type() == "windows":
```

Comment:
changed to double quote? Seems not consistent with other place

Reply:
Yes. It's the same. It's not a big issue, right?!

Reply:
updated code